### PR TITLE
[WIP] Feature/cvsb 19465

### PR DESCRIPTION
**Pull request by JonLazariniDD**
## Ticket title

One line description
[link to ticket number]()

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,154 +59,189 @@ Being a NoSQL database data isn't stored in the relational model, but as a JSON
...
 
 #### Choregraphy
-![Choregraphy](/docs/choregraphy.png      )
 
+![Choregraphy](/docs/choregraphy.png)
 
-In order to understand more about the responsibility of technical records service in the end to end design, please refer to [Ministry Plate Generation Design](https://wiki.dvsacloud.uk/display/HVT/Ministry+Plate+Generation+Design). 
-
-<br>
+In order to understand more about the responsibility of technical records service in the end to end design, please refer to [Ministry Plate Generation Design](https://wiki.dvsacloud.uk/display/HVT/Ministry+Plate+Generation+Design).
```
Comment (bristol-mike-jones) [inline README.md:76]: Is this reference intentional? The service is pretty instrumental to the overall design - I think *most* tasks use it.  I wouldn't want yo single out the plates as the main place - it's used just about everywhere!

File: tests/features/10209.ACs.feature [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-  Scenario: PUT request: HGV vehicle is updated, and the appropriate attributes are automatically set
-    Given I am the vehicles backend service
-    When an existing HGV vehicle is updated via the PUT verb
-    Then my PUT action adheres to the HGV validations, present in the linked excel, columns D-E
-    And a new identical tech record is created, with the same status, and the updated attributes on it
-    And the previous "pre-update" tech record still exists in DynamoDB, with it's status set to archived
-    And the appropriate audit attributes are set on the new updated tech record
-    And I am only able to update attributes within the techRecord[] array
+#Feature: HGV Backend service update | Retrieval + Creation + Update
+#
```
Comment (bristol-mike-jones) [inline tests/features/10209.ACs.feature:2]: ? Why remove these testcases?

File: README.md (hunk 1 above)
Comment (JonLazariniDD) [reply README.md:76]: I agree, it should probably not be part of this README, removing and resolving

**Review by Daniel-Searle — APPROVED** [review]

**Review by m-ujaffer — APPROVED** [review]

**Review by sriniarumugam — APPROVED** [review]
LGTM

**Pull request merged**